Editor/LyumaAv3EditorSupport.cs

```diff
@@ -93,6 +93,27 @@ public static class LyumaAv3EditorSupport
 				// Highlighter.Highlight("Inspector", "Animator To Debug");
 			};
 
+			LyumaAv3Runtime.updateAnimatorWindowDelegate = (rac) =>
+			{
+				if (rac == null)
+				{
+					LyumaAv3Runtime.updateSelectionDelegate(LyumaAv3Emulator.emulatorInstance);
+					return;
+				}
+				if (LyumaAv3Emulator.emulatorInstance.SelectAssetOnChangeAnimatorToDebug)
+				{
+					LyumaAv3Runtime.updateSelectionDelegate(rac);
+					return;
+				}
+
+				var type = System.Type.GetType("UnityEditor.Graphs.AnimatorControllerTool, UnityEditor.Graphs");
+				var prop = type?.GetProperty("animatorController");
+				if (type != null && prop != null)
+				{
+					prop.SetValue(EditorWindow.GetWindow(type, false, "Animator", false), rac);
```

Comment:
Might `EditorWindow.GetWindow` malfunction if the user does not currently have any Animator window open?

https://docs.unity3d.com/ScriptReference/EditorWindow.GetWindow.html says it will create a window if one does not exist, but I'm scared of hitting the glitch where you end up with two animator windows and one says "Failed to Load". Or did unity finally fix that?

Either way, I'm a little scared to pop up a new animator window by default, though maybe it's ok if the user intentionally chooses an "animator to debug"

Reply:
Thank you for the feedback!
I fixed it to check whether the Animator window is already open.

Since there was only a generic version of `EditorWindow.HasOpenInstances`, I used `Resources.FindObjectsOfTypeAll` in a similar way to that method.
https://github.com/Unity-Technologies/UnityCsReference/blob/2022.3/Editor/Mono/EditorWindow.cs#L897